### ISSUE TITLE
Passing search string (not entire where clause) to predicate in search service fails

### ISSUE THREAD

**Issue by stdavis** [agrc/api.mapserv.utah.gov#40]
For example:
https://api.mapserv.utah.gov/api/v1/search/SGID10.BOUNDARIES.Counties/NAME?predicate=salt&apiKey=AGRC-Explorer

Returns:
```json
{
 "status": 400,
 "message": "SGID10.BOUNDARIES.COUNTIES probably does not exist. Check your spelling."
}
```

whereas:
https://api.mapserv.utah.gov/api/v1/search/SGID10.BOUNDARIES.Counties/NAME?predicate=NAME+LIKE+'salt%25'&apiKey=AGRC-Explorer

returns the expected feature

```json
{
 "result": [
  {
   "attributes": {
    "name": "SALT LAKE"
   }
  }
 ],
 "status": 200
}
```

The docs
>Search criteria for finding specific features in featureClass. Any valid ArcObjects where clause will work. If omitted, a TSQL * will be used instead. eg: NAME LIKE 'K%'

read to me as if these two requests should return the same results.

**Comment by steveoh** [comment]
Can you really set a where clause to salt in arc objects and it will work?

**Comment by stdavis** [comment]
No. Maybe we just need to clarify the docs. I read 
>If omitted, a TSQL * will be used instead. eg: NAME LIKE 'K%'
to mean that if you just passed a string then it will automatically create an expression using the NAME field. Probably just my bad reading but I did get an email from Lee Eschler this morning asking about it and he was doing the same thing. The error message saying that the feature class probably doesn't exist doesn't help either.

Maybe the docs would be more clear like this:
>Filter criteria for finding specific features in featureClass. Any valid ArcObjects where clause will work eg: NAME LIKE 'K%'. If omitted, all features will be returned.

**Comment by steveoh** [comment]
Clarifying the doc sounds great. Do you want to take a stab at it and I'll look into the error message?

**Comment by stdavis** [comment]
Closed with #41
